assets/css/styles.css

```diff
@@ -396,3 +403,50 @@ input::placeholder {
     padding-left: 1rem;
   }
 }
+
+@media only screen and (max-width: 414px) {
```

Comment:
Any reason why you selected `414px` here. Since we are using bootstrap for this project, can we align our breakpoint with bootstrap responsive breakpoint? https://getbootstrap.com/docs/4.1/layout/overview/

If a few things are breaking after aligning with the bootstrap responsive breakpoint, you should handle them separately with a comment on the top of the media code explaining why you are adopting the custom size. This will make it easier to understand what each breakpoint are doing.